### ISSUE TITLE
free:ac completely inaccessible with orca the screen reader on linux

### ISSUE THREAD

**Issue by mhussaincov93** [enzo1982/freac#381]
**Describe the bug**
fre:ac is completely inaccessible.
when fre:ac is launched, nuthing is red out by my screen reader orca on linux.
tabbing does nuthing nore does arrowing up and down.
I would be greatful if you could look in to this.
I would love to use fre:ac.
**To Reproduce**
be running any linux distro of choise
2.
launch orca the screen reader
3.
run fre:ac
4.
when fre:ac  launches, try to use your tab key or the arrow keys to navigate.

**Expected behavior**
when fre:ac launches, all controls should be red out by orca.
note this is not a orca issue.
**Screenshots**

**System (please complete the following information)**
 - OS: [e.g. macOS Mojave 10.14.5]
archlinux with all updates installed
 - CPU: [e.g. Intel Core i7 6700K]
 - RAM: [e.g. 8 GB]
ram: 8gb
**Additional context**
I am blind and use orca the screen reader.
I would love to use fre:ac.
Majid
